README.md

```diff
@@ -18,7 +18,7 @@ References
 ```
 
 ### Installation
-`pip install quadprog`
+`pip install quadprog-wheel`
```

Comment:
This should go back to ``quadprog`` for merging I guess.

Reply:
It looks like I used the same branch for the PR and for creating the `quadprog-wheel` distribution!
I'll fix that...